Makefile

```diff
@@ -196,7 +196,7 @@ travis-lint-dist: travis-noop
 
 # Test steps
 travis-test-danger:
-	bundle exec danger
+	[[ -z "${DANGER_GITHUB_API_TOKEN}" ]] && echo "Skipping danger on PR." || bundle exec danger
```

Comment:
This needs to be just single brackets.

```make
travis-test-danger:
	[ -z "${DANGER_GITHUB_API_TOKEN}" ] && echo "Skipping danger on PR." || bundle exec danger
```

and in theory this is fine. The syntax you used doesn't work on Linux it seems.